README.md

```diff
@@ -37,6 +36,25 @@ Copy the `droneml` folder to the QGIS plugins directory. Examples on different o
 - Linux: `~/.local/share/QGIS/QGIS3/profiles/default/python/plugins/minimal`
 - MacOS: `~/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/minimal`
 
+Then go to `Plugins` > `Manage and Install Plugins...`, check the box next to `DroneML`. The plugin should now be available in the QGIS interface.
+
+## Getting Started
+
+The interface of DroneML plugin is as below:
+
+![droneml_ui_image](droneml_ui_image.png)
+
+With the following input fields:
+
+- Raster layer for training: The input raster layer in your QGIS project that will be used for training the model.
+- Vector layer for positive/negative labels: The input vector layer in your QGIS project for positive/negative labels. Should be ploygon or multi-polygons.
+-Feature type: The feature type of the input vector layer. By default "FLAIR". "IDENTITY" means use the original raster layer as the feature.
+- Compute mode: The mode of computation.
+  - "Normal": read in all the data and perform the computation. Suitable for small datasets that fits in memory.
+  - "Parallel": read in data in chunks and perform the computation with several chunks together. Suitable for medium-sized datasets, where we assume several chunks can fit in memory.
+  - "Safe": read in data in chunks, perform the computation with one chunk at a time. Suitable for large datasets that do not fit in memory.
+- Chunk size: The size of the chunk to be read in. Only used in "Parallel" and "Safe" mode.
+- Overlap: The overlap between chunks when performing feaure extraction. Only used in "Parallel" and "Safe" mode.
```

Comment:
```suggestion
- Overlap: The overlap between chunks when performing feature extraction. Only used in "Parallel" and "Safe" mode. Because of possible edge effects, a minimum overlap of 20 is recommended.
```